### DISCUSSION TITLE
How to enable MathJax

### DISCUSSION THREAD

**Discussion by linw789** [KDE/ghostwriter#885] [Q&A] [answered]
As the title explains, I couldn't seem to enable MathJax in Ghostwriter. What's the format for writing math equations? I tried double `$` signs, but it doesn't seem to work.

**Comment by DamienC35** [comment, marked as answer]
Using `$` signs is OK. To get math work, I selected `Pandoc MultiMarkdown` (or simply Pandoc) in the options of previsualisation, instead of `cmark-gfm`.

**Reply by linw789** [reply]
Thanks for the answer. `$` works now after selecting `Pandoc` as previsualization. But `\\` doesn't seem to create a newline in `$$`. Any idea how to make `\\` work?

**Reply by RonKaminsky** [reply]
A possible workaround is to open an "align*" environment between the $$s ("\begin{align*} eq1 \\\\ eq2 \end{align*}").
